apps/laboratory/tests/README.md

```diff
@@ -1,3 +1,14 @@
-# Debugging
+# Functional Tests
+
+We use Playwright as our functional test runner. It's configured to try multiple permutations:
+* Browsers: Chromium/Firefox
+* Modal flavors: default/SIWE/email
+
+## Running Tests
+
+* `npx playwright test` to run in default mode (make sure your `.env` is set up)
```

Comment:
Should we add mention for `MAILSAC_API_KEY` env variable here as well?

Reply:
`MAILSAC_API_KEY` is in `.env.example` so I don't think it needs an extra mention.